cmd/operator/main.go

```diff
@@ -114,8 +115,9 @@ func init() {
 
 	flag.StringVar(&watchInterval, "watch-interval", defaultWatchInterval, "Interval for polling pods' state, e.g. '2m'. Set to '0' to disable, or 'once' to only run once when the operator starts")
 	flag.IntVar(&maxPodRestartsPerMinute, "max-pod-restarts-per-minute", defaultMaxPodRestartsPerMinute, "Maximum number of pods in an invalid state that can be restarted per minute")
-	flag.BoolVar(&disableLeaderElection, "disable-leader-election", false, "Disable leader election for operator")
 
+	flag.BoolVar(&disableLeaderElection, "disable-leader-election", false, "Disable leader election for operator")
+	flag.BoolVar(&disableServiceReconciler, "disable-service-reconciler", false, "Disable the Service reconciler for Dapr-enabled Deployments and StatefulSets")
```

Comment:
nitpick: if grouping can group Bool vars together.

Reply:
They're currently grouped by kind. These 3 are all about configuring the reconciler